packages/@ember/-internals/metal/lib/property_get.ts

```diff
@@ -194,6 +194,7 @@ export function getWithDefault<T extends object, K extends Extract<keyof T, stri
     {
       id: 'ember-metal.get-with-default',
       until: '4.0.0',
+      url: 'https://deprecations.emberjs.com/v3.x#toc_ember-metal-get-with-default',
```

Comment:
The link doesn't work just yet, do we expect it to be live already?

Reply:
Oh I see, I just merged https://github.com/ember-learn/deprecation-app/pull/600. Once that is published and we can validate the URL is working as expected this should good to go...

Reply:
Works now! :)